gst/nnstreamer/elements/gsttensor_trainer.c

```diff
@@ -60,6 +60,11 @@ G_DEFINE_TYPE (GstTensorTrainer, gst_tensor_trainer, GST_TYPE_ELEMENT);
  * @brief Default framework property value
  */
 #define DEFAULT_PROP_FRAMEWORK "nntrainer"
+#define DEFAULT_PROP_INPUT_LIST 1
+#define DEFAULT_PROP_LABEL_LIST 1
+#define DEFAULT_PROP_TRAIN_SAMPLES 0
+#define DEFAULT_PROP_VALID_SAMPLES 0
+#define DEFAULT_PROP_EPOCHS 1
```

Comment:
How about setting the default epochs to -1 and repeating it infinitely in this case?

Reply:
I am considering it while creating TC.
I'll update it in the next PR.
Thank you.